src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -0,0 +1,11 @@
+databaseChangeLog:
+#  - includeAll:
+#    path: "db/changelog/0.1"
+#    path: "0.1"
+#    relativeToChangeLogFile: true
+#    errorIfMissingOrEmpty: true
+# выше закомментированный код не работает, никак, день убил на это...
```

тут не очень понятно, из-за того, что он закомментирован, но проблема может быть в том, что includeAll и последующие директивы находятся на одном уровне (по отступам):
```suggestion
   - includeAll:
       path: "db/changelog/0.1"
       path: "0.1"
       relativeToChangeLogFile: true
       errorIfMissingOrEmpty: true
```